### PR TITLE
crontab: refresh

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/crontab.md [hunk 1]
```diff
@@ -28,9 +27,9 @@
 
 `0 10 * * * {{command_to_execute}}`
 
-- Sample job which runs every minute on the 3rd of April:
+- Sample job which runs every 10 minutes:
 
-`* * 3 Apr * {{command_to_execute}}`
+`*/10 * * * * {{command_to_execute}}`
```
Comment (marchersimon) [inline pages/common/crontab.md:32]: I don't really get what this example shows. This isn't a valid command right?

Comment (CleanMachine1) [reply pages/common/crontab.md:32]: There are already pre-existing versions of these commands, but no, you cannot run this command. It is just the format which should be used within the crontab command.

Comment (marchersimon) [reply pages/common/crontab.md:32]: How about a description like `Sample crontab entry, which runs a command every 10 minutes:`?

**Comment by marchersimon** [comment]
I just saw, that <https://manned.org/man/crontab.1> might be a better choice than <https://manned.org/man/crontab>. Not sure tough.

File: pages/common/crontab.md [hunk 2]
```diff
@@ -28,9 +27,9 @@
 
 `0 10 * * * {{command_to_execute}}`
 
-- Sample job which runs every minute on the 3rd of April:
+- Sample job which runs every 10 minutes:
```
Comment (CleanMachine1) [inline pages/common/crontab.md:30]:
```suggestion
- Sample crontab entry, which runs a command every 10 minutes:
```

File: pages/common/crontab.md [hunk 3]
```diff
@@ -28,9 +27,9 @@
 
 `0 10 * * * {{command_to_execute}}`
 
-- Sample job which runs every minute on the 3rd of April:
+- Sample crontab entry, which runs a command every 10 minutes:
 
-`* * 3 Apr * {{command_to_execute}}`
+`*/10 * * * * {{command_to_execute}}`
 
 - Sample job which runs a certain script at 02:30 every Friday:
```
Comment (marchersimon) [inline pages/common/crontab.md:34]: The same applies here
```suggestion
- Sample crontab entry, which runs a certain script at 02:30 every Friday:
```

**Review by marchersimon — APPROVED** [review]
Thanks! 👍🏽

**Pull request merged**